openshift/_python36.dc.json

```diff
@@ -642,15 +617,7 @@
             ],
             "restartPolicy": "Always",
             "dnsPolicy": "ClusterFirst",
```

Comment:
Reminder: Create strategy to migrate data over the PVC's

Reply:
Unsure about this one. The MDS API will no longer connected  to this PVC, it will be mounted to the DocMan instead. Please come chat if we need to hash this out.

Reply:
Smart! I thought there will be a new PVC, not just re-mounting the existing PVC 💯